### DISCUSSION TITLE
Добрый день, не отображается на телефоне VKWebAppShowSlidesSheetResult онбординг

### DISCUSSION THREAD

**Discussion by GithubGapPoint** [VKCOM/VKUI#4677] [Q&A] [answered]
Помогите пожалуйста,  в интернете ничего не получается найти

**Comment by SevereCloud** [comment, marked as answer]
Привет, VKUI это библиотека компонентов, а [VKWebAppShowSlidesSheet](https://dev.vk.com/bridge/VKWebAppShowSlidesSheet) относится к [vk-bridge](https://github.com/VKCOM/vk-bridge).

Чтобы помочь, нужно больше информации о том, как вызывается событие(код). Не вызывается ли `VKWebAppShowSlidesSheetResult` вместо `VKWebAppShowSlidesSheet`

Если это похоже на баг, то не помещает информация о телефоне, версия браузера и клиента VK.

**Reply by GithubGapPoint** [reply]
const shopOnboarding = () => {

	bridge.send('VKWebAppShowSlidesSheet', {
		slides: [
		  {
			media: {
			  blob: тут изображение с ним все хорошо
			  type: 'image'
			},
			title: 'Расписание!',
			subtitle: 'Вас привествует Расписание!'
		  },
		  {
			media: {
			  blob: тут изображение с ним все хорошо
			  type: 'image'
			},
			title: 'Расписание!',
			subtitle: 'Вас привествует Расписание!'
		  }
		 ]})
		.then((data) => { 
		  if (data.result) {
			// Слайды показаны
		  }
		})
		.catch((error) => {
		  // Ошибка
		  console.log(error);
		});
	}
	useEffect(() => {
		async function fetchData() {
			const user = await bridge.send('VKWebAppGetUserInfo');
			setUser(user);
			setPopout(null);
			shopOnboarding();
		}
		fetchData();
	}, []);

**Reply by SevereCloud** [reply]
Вот [пример](https://codesandbox.io/p/sandbox/vkui-vite-cssm-forked-rztdng) и он работает

<details><summary>Скриншот</summary>

![image](https://user-images.githubusercontent.com/14944123/229451555-2c06242b-afd2-49c0-990d-09daa258cae9.png)

</details> 

---

Отображается ли онбординг на десктопной версии? Точно с изображениями все хорошо?
